setup.py

```diff
@@ -1,12 +1,5 @@
 import sys
-from distutils.core import setup
-
-if sys.version_info[:2] < (2, 5) or sys.version_info[0] > 2:
-    msg = ("LabJackPython requires Python 2.5 or later but does not work on "
-           "any version of Python 3.  You are using version %s.  Please "
-           "install using a supported version." % sys.version)
-    sys.stderr.write(msg)
-    sys.exit(1)
+from setuptools import setup
```

Comment:
Yup. Using wheel involves a new dependency on setuptools. Not sure if you are ok with it. At least we have setuptools if you have Python 2 >=2.7.9 or Python 3 >=3.4.